data/strings/types_strings.txt

```diff
@@ -25740,7 +25500,8 @@
     ref = type.tourism.attraction
 
   [type.tourism.camp_site]
-    en = Camp Site
+    en = Campsite
+    en-US = Campground
```

Comment:
A campground is an area or facility which contains multiple camp sites and shared amenities which could include bathrooms, an office to register, or a notice board.

Reply:
campground in en-US is the same thing as campsite in en-GB. Given that only the US calls them campgrounds, I think it's best to keep this as it is.
(campground can be used when en-US and en-generic is implemented :)

Reply:
There is no support yet for en-US, please replace it with en everywhere, and use en-GB for the non-US version.

Reply:
@endim8 can you please use en = Campground and en-GB = Campsite ?